src/utils.jl

```diff
@@ -19,3 +19,19 @@ struct SimpleEdgeVector{V, G <: LG.AbstractGraph{V}} <: AbstractVector{LG.Edge{V
     graph::G
 end
 
+"""
+    OutNeighborVector <: AbstractVector
+A structure for iterating over the out neighbors in a graph for a certain vertex.
```

Comment:
Newline for a bit of space between code and text

```suggestion

A structure for iterating over the out neighbors in a graph for a certain vertex.
```

Reply:
done